### PR TITLE
TASK: Add core migration

### PR DESCRIPTION
**Pull request by gerhard-boden**
Adjusts code to package renaming from "TYPO3.TYPO3CR.Search" to
"Neos.ContentRepository.Search"

### CODE REVIEW

File: Migrations/Code/Version20161210231100.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        return 'Neos.ContentRepository.Search-20161210231100';
+    }
+
+    /**
+     * @return void
+     */
+    public function up()
+    {
+        $this->searchAndReplace('TYPO3\TYPO3CR\Search', 'Neos\Flow');
```
Comment (kdambekalns) [inline Migrations/Code/Version20161210231100.php]: Is this really correct? That replacement looks rather generic…

Comment (gerhard-boden) [inline Migrations/Code/Version20161210231100.php]: damn... thanks a lot for seeing this.. Should not write migrations late at night..

**Comment by gerhard-boden** [comment]
@kdambekalns fixed

**Comment by kitsunet** [comment]
Should go into 2.0 branch now

**Comment by gerhard-boden** [comment]
@kitsunet done!

**Comment by gerhard-boden** [comment]
We should merge this one soonish. Tests failing... maybe requiring wrong depenendencies for tests?

**Pull request merged**